learnrl/playground.py

```diff
@@ -356,19 +359,30 @@ def run(self,
                     callbacks.on_steps_cycle_end(step, logs)
 
                 step += 1
-
+                logs['cur_step'] += 1
+                if steps >= 0 and logs['cur_step'] >= steps:
+                    break
+            
             callbacks.on_episode_end(episode, logs)
 
             if (episode + 1) % episodes_cycle_len == 0 or episode + 1 == episodes:
                 callbacks.on_episodes_cycle_end(episode, logs)
 
+            if steps >= 0 and logs['cur_step'] >= steps:
+                print("Training finished")
+                print("Total steps", logs['cur_step'])
+                break   
+            
```

Comment:
This should not exist, print statements are unwanted outside of a verbose parameter.
Also this could break the logger behavior (unchecked).